docs/UserGuide.md

```diff
@@ -3,7 +3,7 @@ layout: page
 title: MediBase3 User Guide
 ---
 
-MediBase3 (MB3) is a **desktop app for doctors, offering an all in one solution to manage their patient records with ease.** Be it medical conditions, allergies, priorities or even appointments you can to leave it for MB3 to handle and focus more on your patients! 
+MediBase3 (MB3) is a **desktop app for doctors, offering an all in one solution to manage his/her patient records with ease.** Be it medical conditions, allergies, priorities or even appointments you can to leave it for us to handle and focus more on your patients! 
```

Comment:
i think its fine to keep `their` in this case?

Reply:
Is it OK to drop the his/her

"allow doctors to manage patient records"

Reply:
Yea i think that would be better

Reply:
I don't think it should be "can to leave it" and I think can keep 'MB3' over 'us' as it make it sound like the developers are the one who will handle

Reply:
For bottom portion change it to "leave it to MB3"